File: TransCoder_tokenized_test_set_functions/java/CALCULATE_SUM_OF_ALL_NUMBERS_PRESENT_IN_A_STRING.java
static int findSum ( String str ) {
  String temp = "";
  int sum = 0;
  for ( int i = 0;
  i < str . length ( );
  i ++ ) {
    char ch = str . charAt ( i );
    if ( Character . isDigit ( ch ) ) temp += ch;
    else {
      sum += Integer . parseInt ( temp );
      temp = "0";
    }
  }
  return sum + Integer . parseInt ( temp );
}

File: TransCoder_tokenized_test_set_functions/cpp/CALCULATE_SUM_OF_ALL_NUMBERS_PRESENT_IN_A_STRING.cpp
int findSum ( string str ) {
  string temp = "";
  int sum = 0;
  for ( char ch : str ) {
    if ( isdigit ( ch ) ) temp += ch;
    else {
      sum += atoi ( temp . c_str ( ) );
      temp = "";
    }
  }
  return sum + atoi ( temp . c_str ( ) );
}